funnel/forms/space.py

```diff
@@ -92,6 +93,30 @@ class ParticipantForm(forms.Form):
         validators=[forms.validators.DataRequired(u"Select at least one event")])
 
 
+# TODO: Move to Baseframe
+class ValidFile(object):
+    def __init__(self, allowed_exts=[], message=None):
+        if not message:
+            message = __(u"Please upload a valid file.")
+        self.message = message
+        self.allowed_exts = allowed_exts
+
+    def __call__(self, form, field):
+        ext = os.path.splitext(field.data.filename)[1].strip(".")
+        if ext and ext.lower() not in self.allowed_exts:
+            raise wtforms.validators.StopValidation(self.message)
+
+
+class ParticipantImportForm(forms.Form):
+    participant_list = forms.FileField(__("Participant list"),
+        description=u"Expected Attributes: name, email, phone, twitter, company.",
```

Comment:
Also, if the source doesn't have a twitter header, import shouldn't fail. Minimum requirements are only name and email.